packages/office-ui-fabric-react/src/components/Divider/Divider.Props.ts

```diff
@@ -0,0 +1,29 @@
+export interface IDivider {
+
+}
+
+export interface IDividerProps {
+
+  /**
+   * Optional callback to access the IDivider interface. Use this instead of ref for accessing
+   * the public methods and properties of the component.
+   */
+  componentRef?: (component: IDivider) => void;
+
+  /**
+   * The height of the divider.
```

Comment:
So I debated whether the customization hook should be through a classNames prop or explicit properties here. I went with explicit properties because there isn't really a "default divider", it is very much context specific, so you'd be specifying these styles every time. 

Open to feedback on it.